tests/unit/utils/test_files.py

```diff
@@ -122,6 +116,13 @@ def test_images_supported(mocker):
     assert not directories
 
 
+def test_order():
```

Comment:
We may want to mock the api setting here

Reply:
no longer needed in the new implementation, as sorting is done in the setting directly